### ISSUE TITLE
Весь код ангуляра одним компонентом

### ISSUE THREAD

**Issue by IshchenkoRomanWork** [otetsdiodor/CoreBattle#39]
https://github.com/otetsdiodor/CoreBattle/blob/f48869ada464c00ad88d3725b043acc78ec4d8b2/CoreBattle/ClientApp/src/app/game/game.component.ts#L1-L138

**Comment by otetsdiodor** [comment]
Да, осознанное решение, что бы не грузить прект не нужными компонентами
